app/Http/Controllers/Bible/BiblesController.php

```diff
@@ -287,6 +287,59 @@ public function search($search_text)
         return $this->reply($bibles);
     }
 
+    /**
+     * Description:
+     * Return the bible data by given Bible ID.
+     *
+     * @OA\Get(
+     *     path="/bibles/search",
+     *     tags={"Bibles"},
+     *     summary="Returns metadata for all bibles meeting the given bible_id in it's Bible ID",
+     *     description="metadata for all bibles meeting the bible_id in it's Bible ID",
+     *     operationId="v4_bible_by_id.search",
+     *     @OA\Parameter(name="bible_id",in="query",required=true,@OA\Schema(ref="#/components/schemas/Bible/properties/id")),
+     *     @OA\Response(
+     *         response=200,
+     *         description="successful operation",
+     *         @OA\MediaType(mediaType="application/json",
+     *         @OA\Schema(ref="#/components/schemas/v4_bible.search"))
+     *     )
+     * )
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function searchByBibleId()
```

Comment:
throughout, the correct term is bible version, not bible id. The bible id contains the language. 
so this should change to searchByBibleVersion
I won't comment further on this, but expect the variable names, etc to be changed throughout

Reply:
@bradflood, Ok, I agree. I have changed to `searchByBibleVersion`. Also, I have changed the query parameter name to `version`.